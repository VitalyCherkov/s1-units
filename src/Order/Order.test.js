import React from 'react';
import {shallow, configure} from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

import Order from './Order';

const defaultProps = {
  shop: '@SHOP@',
  items: [
    '@ITEM1',
    '@ITEM2',
    '@ITEM3',
  ],
  date: 123,
};

configure({ adapter: new Adapter() });

describe('App component', () => {
  const wrapper = shallow(<Order order={defaultProps} />);

  it('', () => {
    const order = defaultProps;

    wrapper.setProps({ order });
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
