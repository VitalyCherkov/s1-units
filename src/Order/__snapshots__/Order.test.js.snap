// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component  1`] = `
<div
  className="Order"
>
  <div
    className="Order-header"
  >
    <span
      className="Order-shop"
    >
      @SHOP@
    </span>
    <span>
      1 января, чт, 1970 год
    </span>
  </div>
  <div
    className="Order-items"
  >
    <div
      className="Order-item"
      key="0"
    >
      @ITEM1
    </div>
    <div
      className="Order-item"
      key="1"
    >
      @ITEM2
    </div>
    <div
      className="Order-item"
      key="2"
    >
      @ITEM3
    </div>
  </div>
</div>
`;
